examples/front_ends/simple_auth/docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3.8'
+
+services:
+  oauth2-server:
+    build: .
+    ports:
+      - "5000:5000"
```

Comment:
This fails to bind the port on macOS due to 5000 being reserved for AirPlay.

This is why we needed the configurable port.